<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSoftUpgredeWidget</class>
 <widget class="QWidget" name="CSoftUpgredeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSoftUpgredeWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>品牌:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_brand">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>选择品牌</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>添加子品牌：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_brand">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>输入品牌名称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbtn_addbrand">
       <property name="toolTip">
        <string>添加品牌</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="CSMonitorServer.qrc">
         <normaloff>:/Resources/plus.png</normaloff>:/Resources/plus.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbtn_Passwd">
       <property name="toolTip">
        <string>品牌密钥配置</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="CSMonitorServer.qrc">
         <normaloff>:/Resources/key.png</normaloff>:/Resources/key.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectItems</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>软件名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>版本号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Url</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>软件品牌</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>软件类别</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>存储位置</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>主程序</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QToolButton" name="tbtn_add">
       <property name="toolTip">
        <string>添加软件</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="CSMonitorServer.qrc">
         <normaloff>:/Resources/plus.png</normaloff>:/Resources/plus.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbtn_delete">
       <property name="toolTip">
        <string>删除</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="CSMonitorServer.qrc">
         <normaloff>:/Resources/minus.png</normaloff>:/Resources/minus.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Init">
       <property name="text">
        <string>还原</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_SaveData">
       <property name="text">
        <string>保存配置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Save">
       <property name="text">
        <string>生成 XML</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CSMonitorServer.qrc"/>
 </resources>
 <connections/>
</ui>
